code/modules/power/lighting.dm

```diff
@@ -218,27 +217,23 @@
 	return FALSE
 
 /obj/machinery/light/update_icon()
-
 	switch(status)		// set icon_states
-		if(LIGHT_OK)
-			icon_state = "[base_state][on]"
+		if(LIGHT_OK, LIGHT_DISABLED)
+			icon_state = "[base_state][light_on]"
 		if(LIGHT_EMPTY)
 			icon_state = "[base_state]-empty"
-			on = FALSE
 		if(LIGHT_BURNED)
 			icon_state = "[base_state]-burned"
-			on = FALSE
 		if(LIGHT_BROKEN)
 			icon_state = "[base_state]-broken"
-			on = FALSE
 	return
 
 // update the icon_state and luminosity of the light depending on its state
 /obj/machinery/light/proc/update(trigger = TRUE)
 	switch(status)
-		if(LIGHT_BROKEN, LIGHT_BURNED, LIGHT_EMPTY)
-			on = FALSE
-	if(on)
+		if(LIGHT_BROKEN, LIGHT_BURNED, LIGHT_EMPTY, LIGHT_DISABLED)
+			light_on = FALSE
+	if(light_on)
```

Comment:
Change this back:
Light_on is specifically supposed to be a movable lights var so try to keep the systems seperate